<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BorrowRecordWindow</class>
 <widget class="QDialog" name="BorrowRecordWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>借阅记录</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="tableWidgetRecords">
     <column>
      <property name="text">
       <string>记录ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>书号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>书名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>借阅日期</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>应还日期</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>归还状态</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>逾期天数</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>借阅人</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelOverdue">
       <property name="text">
        <string>逾期记录：0 条</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRefresh">
       <property name="text">
        <string>刷新</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonReturn">
       <property name="text">
        <string>归还所选记录</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
